### PR TITLE
Change note.txt version 2

### PR DESCRIPTION
**Pull request by TechMarDay**
Change note.txt version 2

### CODE REVIEW

**Review by TechMarDay — COMMENTED** [review]
approved

**Comment by TechMarDay** [comment]
ok

File: Note.txt [hunk 1]
```diff
@@ -1 +1 @@
-test note file
```
Comment (TechMarDay) [inline Note.txt:1]: ok

**Review by TechMarDay — COMMENTED** [review]
test

**Review by LapTrinhThucChien — APPROVED** [review]
approved

**Review by LapTrinhThucChien — APPROVED** [review]
approved

File: Note.txt (hunk 1 above)
Comment (LapTrinhThucChien) [reply Note.txt:1]: APPROVED

**Review by LapTrinhThucChien — APPROVED** [review]

**Review by LapTrinhThucChien — APPROVED** [review]
S

**Pull request merged**